pyEpiabm/pyEpiabm/sweep/intervention_sweep.py

```diff
@@ -72,9 +86,20 @@ def __call__(self, time):
             num_cases = sum(map(lambda cell: cell.number_infectious(),
                             self._population.cells))
             if intervention.is_active(time, num_cases):
-                intervention(time)
                 if self.intervention_active_status[intervention] is False:
+                    # Update with the single intervention
```

Comment:
Can you make this more informative? What are you updating here? And what information are you adding?